api/user.go

```diff
@@ -588,8 +591,13 @@ func getAudits(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-func createProfileImage(username string, userId string) ([]byte, *model.AppError) {
+var (
+	dpi      = flag.Float64("dpi", 72, "screen resolution in Dots Per Inch")
```

Comment:
Ok yeah that's fine, I'll just set it without using flags, don't think it needs to be configurable (plus from what I can tell it would be hard to get it to look good consistently if we allowed things like font size to be changed)